### PR TITLE
Adding Windows DPI awareness support and editorconfig.

### PR DESCRIPTION
**Pull request by Slion**
Not sure how you guys are handling contributions, hope you like that one. I'm probably missing some library include in other configs. Should CMake be able to target Windows?

### CODE REVIEW

**Comment by seanpaultaylor** [comment]
Shouldn't we always want to be dpi aware ? Why do we need config for this ?

**Comment by Slion** [comment]
An app designed for low res would not want to run at that low res on a high DPI display so you'll not want to make it DPI aware. Thus it gets stretched and is still usable though not looking as good as it could.

**Comment by seanpaultaylor** [comment]
If we add this feature, it should be done on mac (retina) and linux too.

**Comment by seanpaultaylor** [comment]
Please submit to next branch not master.

File: gameplay/gameplay.vcxproj [hunk 1]
```diff
@@ -629,6 +629,7 @@
     </Link>
     <Lib>
       <TargetMachine>MachineX64</TargetMachine>
+      <AdditionalDependencies>  shcore.lib</AdditionalDependencies>
```
Comment (seanpaultaylor) [inline gameplay/gameplay.vcxproj]: Remove extra space

**Pull request closed without merging**